OpenRA.Game/Exts.cs

```diff
@@ -415,7 +415,7 @@ public static IEnumerable<T> Exclude<T>(this IEnumerable<T> ts, params T[] exclu
 
 		public static HashSet<T> ToHashSet<T>(this IEnumerable<T> source)
 		{
-			return new HashSet<T>(source);
+			return [.. source];
 		}
```

Comment:
This functions should just be removed right?

Reply:
Indeed it's been added to the BCL, removed.